pkg/webhook/webhook_test.go

```diff
@@ -1151,7 +1151,7 @@ func TestInjectProxySidecarContainer(t *testing.T) {
 	m := &podMutator{proxyImage: imageURL}
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			containers := m.injectProxySidecarContainer(test.containers, proxyPort)
+			containers := m.injectProxySidecarContainer(test.containers, proxyPort, nil)
```

Comment:
Update this table test to include restart policy and ensure the return result contains the desired restart policy